src/OrbitGl/GlCanvas.cpp

```diff
@@ -27,7 +27,8 @@ float GlCanvas::kZValueTrack = 0.01f;
 float GlCanvas::kZValueEventBar = 0.03f;
 float GlCanvas::kZValueBox = 0.05f;
 float GlCanvas::kZValueEvent = 0.07f;
-float GlCanvas::kZValueTrackText = 0.09f;
+float GlCanvas::kZValueTrackText = 0.08f;
+float GlCanvas::kZValueTrackLabel = 0.09f;
```

Comment:
Can I also ask you to modify line 46-48 with...
```
constexpr unsigned kNumberOriginalLayers = 17;
constexpr unsigned kExtraLayersForMovingTracks = 6;
constexpr unsigned kExtraLayersForPinnedTracks = 6;
```
?

Reply:
Done! :-)

Reply:
Not your problem, we should change this manually updated numbers :-)